lib/shared/drive-constraints.js

```diff
@@ -208,3 +208,115 @@ exports.isDriveValid = (drive, image) => {
 exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE);
 };
+
+/**
+ * @summary Drive/image compatibility status messages.
+ * @public
+ *
+ * @description
+ * Status messages intended to be shown to the user.
+ *
+ * @type {Object}
+ */
+exports.COMPATIBILITY_STATUS_MESSAGES = {
```

Comment:
Maybe we can descriptions of what each status means, like in https://github.com/resin-io/etcher/blob/master/lib/shared/exit-codes.js#L24? For example, that "locked" means an SD Card locking (vs something else), etc?